docs/automated-tests/test-execution/partial-test-execution.md

```diff
@@ -1,69 +1,75 @@
 
 ![Run To here][1]
 
-The triggered partial run executes the preceding (including the selected) test step(s). Once the step execution finishes, the __executing browser remains opened, presenting the current state of the application under test, and a recorder gets attached to it__.
+The triggered partial run executes the preceding and the selected test steps. Once the step execution finishes, the executing browser stays opened, presents the current state of the tested application, and gets a recorder attached.
 
 ![Run To here finished][2]
 
 > __Tip__
 > <br>
 > <br>
-> The __attached recorder to the browser is enabled for recording__. Thus any actions against the page will be captured and recorded in the active test.
+> The recorder attached to the browser is enabled, captures all actions against the page, and then records them in the active test.
 > <br>
-> Ensure to __pause the recorder__, if you need to debug your test.
+> To debug the test, pause the recorder.
 
-Although the run is only partial, it generates a complete execution log for the executed steps. And in case any of the executed steps failed, before the selected one was reached, the failure details are listed in the execution log and the recorder still gets attached to the browser.
+The partial test run generates a complete execution log. If an executed step fails before the test reaches the selected to-here step, the execution log lists the failure details, and the recorder still gets attached to the browser.
 
 ## Run From Here
 
-The __Run From Here__ option is only available in the __context of an active recording session__ and a single step selected from a test. It is also listed in the _Run..._ sub-menu. It executes the subsequent steps starting from the selected one in an existing browser instance with attached recorder.
+The __From Here__ partial test is available only when you select a single test step. The partial test executes the selected step and all subsequent steps. The partial test runs in an existing browser instance with an attached recorder.
 
 ![Run From here][3]
 
-It generates complete execution log, so in case any of the executed steps fails, the failure details are listed in the execution log and the recorder remains attached to the browser.
+The partial test run generates a complete execution log. If an executed step fails, the execution log lists the failure details, and the recorder stays attached to the browser.
 
 ![Run From here finished][4]
 
 ## Run Selected Steps
 
-The option to execute few steps selected from a test, __Run Selected Steps__, is the last listed in the _Run..._ sub-menu. It is only applicable if there is an __active recording session and one or more, but subsequent, steps are selected__. These steps gets executed in the order they are listed in the test.
```

Comment:
Don't use "there is/there are". You can almost always replace this expression with something more concise.